bin/metrics-rabbitmq-queue.rb

```diff
@@ -48,9 +49,9 @@ class RabbitMQMetrics < Sensu::Plugin::Metric::CLI::Graphite
          short: '-v',
          long: '--vhost VHOST'
 
-  option :user,
+  option :username,
```

Comment:
I like making it consistent but this makes it a breaking change. This will need to be called out in the CHANGELOG.